.dockerignore

```diff
@@ -0,0 +1 @@
+Dockerfile
```

Comment:
It's good idea to add to this file:
site/
target/
circom/target/
circom_algebra/target/
compiler/target/
constraint_generation/target/
parser/target/
program_structure/target/
type_analysis/target/
.idea/
.DS_Store
.git

Reply:
Added, thanks!